pom.xml

```diff
@@ -68,6 +68,10 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Good point, It doesn't work directly because of this:
```
[ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:3.2.1:test-run (default-cli) on project spring-petclinic: Execution default-cli of goal org.springframework.boot:spring-boot-maven-plugin:3.2.1:test-run failed: Unable to find a single main class from the following candidates [org.springframework.samples.petclinic.PostgresIntegrationTests, org.springframework.samples.petclinic.PetClinicIntegrationTests, org.springframework.samples.petclinic.MysqlTestApplication]
```
but I've modified the PR to keep the docker compose dependency in test scope.